### PR TITLE
Shrink dt range for valid data

### PR DESCRIPTION
**Pull request by calcmogul**

### CODE REVIEW

File: src/integrationtest/native/cpp/SimulationTest.cpp [hunk 1]
```diff
@@ -119,8 +119,8 @@ class IntegrationTest : public ::testing::Test {
       if (m_settings.mechanism == sysid::analysis::kArm) {
         // The acceleration fit (used only for arms) is worse than the next
         // velocity fit
-        EXPECT_NEAR(Kv, ffGains[1], 0.015);
-        EXPECT_NEAR(Ka, ffGains[2], 0.04);
+        EXPECT_NEAR(Kv, ffGains[1], 0.01);
+        EXPECT_NEAR(Ka, ffGains[2], 0.01);
```
Comment (Piphi5) [inline src/integrationtest/native/cpp/SimulationTest.cpp:123]: How many tests have you done with these values?

Comment (calcmogul) [reply src/integrationtest/native/cpp/SimulationTest.cpp:123]: Two, including this one.

Comment (Piphi5) [reply src/integrationtest/native/cpp/SimulationTest.cpp:123]: Looks like we need to increase kA?

Comment (calcmogul) [reply src/integrationtest/native/cpp/SimulationTest.cpp:123]: It's off by like 0.07. :/

Comment (Piphi5) [reply src/integrationtest/native/cpp/SimulationTest.cpp:123]: Mac is cursed

**Comment by calcmogul** [comment]
Locally, using the 1ms dt band makes it more accurate. It seems to be a tradeoff between noise magnitude and datapoint density.

**Pull request closed without merging**